### PR TITLE
fix: Realtime docs

### PR DESCRIPTION
**Pull request by chasers**
- [x] Makes it more clear what the rate limits are
- [x] Changes "rate limits" to "quotas"
- [x] Fixes custom jwt usage example per https://github.com/supabase/supabase-js/issues/553#issuecomment-1531182767

### CODE REVIEW

**Comment by chasers** [comment]
@w3b6x9 just wanted to bring this to your attention. Let's discuss when you're back.

File: apps/docs/pages/guides/realtime/extensions/postgres-changes.mdx [hunk 1]
```diff
@@ -352,10 +352,10 @@ const { createClient } = require('@supabase/supabase-js')
 const supabase = createClient(process.env.SUPABASE_URL, process.env.SUPABASE_KEY, {
   realtime: {
     headers: {
-      apikey: 'supabase-signed-token',
+      apikey: `Bearer ${your_custom_token}`,
```
Comment (gregnr) [inline apps/docs/pages/guides/realtime/extensions/postgres-changes.mdx:355]: Confirming this should be `Bearer` for `apikey` (vs. `authorization`)?

Comment (chasers) [reply apps/docs/pages/guides/realtime/extensions/postgres-changes.mdx:355]: Yeah it's odd, but it's what seems to work. I'd like to figure out exactly why.

Comment (gregnr) [reply apps/docs/pages/guides/realtime/extensions/postgres-changes.mdx:355]: Interesting - no `Bearer` prefix for non-realtime endpoints - agreed would be good to understand this

**Review by gregnr — APPROVED** [review]

File: apps/docs/pages/guides/realtime/rate-limits.mdx [hunk 2]
```diff
@@ -11,23 +11,48 @@ Supabase Realtime is a global cluster. We've implemented some rate limits to hel
 
 Rate limits are configurable per project and our cluster supports millions of concurrent connections. [Contact support](https://app.supabase.com/support/new) if these limits are causing issues.
 
-## Limits by Plan
+## Rate Limits by Plan
 
-Limits are set on the Free and Pro plans accordingly. See our [pricing page](https://supabase.com/pricing) for more information.
+Rate limits are different than usage limits. The usage limits for Free and Pro projects can be found on the See our [pricing page](https://supabase.com/pricing).
```
Comment (w3b6x9) [inline apps/docs/pages/guides/realtime/rate-limits.mdx:16]: @chasers we're making the distinction between rate and usage limits here. can we add in a brief explanation for each to demystify them?

Comment (w3b6x9) [inline apps/docs/pages/guides/realtime/rate-limits.mdx:16]: @chasers 

```suggestion
Rate limits are different than usage limits. The usage limits for Free and Pro projects can be found on our [pricing page](https://supabase.com/pricing).
```

File: apps/docs/pages/guides/realtime/rate-limits.mdx [hunk 3]
```diff
@@ -11,23 +11,48 @@ Supabase Realtime is a global cluster. We've implemented some rate limits to hel
...
 
-Enterprise plans are billed on a usage basis. We still employ limiting on Enterprise plans. If you're on an Enterprise plan simply contact support and we'll increase your limits as needed.
+Rate limits protect our infrasructure. You can increase your rate limits by upgrading your plan and removing your spend cap.
```
Comment (w3b6x9) [inline apps/docs/pages/guides/realtime/rate-limits.mdx:18]: @chasers

```suggestion
Rate limits protect our infrastructure. You can increase your rate limits by upgrading your plan and removing your spend cap.
```

Comment (w3b6x9) [inline apps/docs/pages/guides/realtime/rate-limits.mdx:18]: @chasers we should position this as rate limits exist to ensure you're not unknowingly overspending and to make sure that Realtime stays performant under expected usage.

File: apps/docs/pages/guides/realtime/rate-limits.mdx [hunk 4]
```diff
@@ -11,23 +11,48 @@ Supabase Realtime is a global cluster. We've implemented some rate limits to hel
...
 
-Enterprise plan limits start at:
+If you need your rate limits increased further please [contact support](https://app.supabase.com/support/new).
```
Comment (w3b6x9) [inline apps/docs/pages/guides/realtime/rate-limits.mdx:20]: @chasers you mentioned usage limits briefly above. We should mention that folks can reach out to increase usage limits as well given we explain what rate and usage limits are (see: https://github.com/supabase/supabase/pull/14322/files#r1220626756).

File: apps/docs/pages/guides/realtime/rate-limits.mdx [hunk 5]
```diff
@@ -11,23 +11,48 @@ Supabase Realtime is a global cluster. We've implemented some rate limits to hel
...
+- 1,000 Channel joins per second
+
+### Enterprise
+
+Starts at:
+
+- 5,000 concurrent clients
 - 1,000 messages per second
-- 500 concurrent Channels
+- 1,000 Channel joins per second
```
Comment (w3b6x9) [inline apps/docs/pages/guides/realtime/rate-limits.mdx:50]: @chasers we should just add these to the Realtime pricing section at this point. feels weird to have these in here and not listed out on pricing page.

File: apps/docs/pages/guides/realtime/rate-limits.mdx [hunk 6]
```diff
@@ -11,23 +11,48 @@ Supabase Realtime is a global cluster. We've implemented some rate limits to hel
...
 
 ## System Limits
 
 The following limits apply to all projects:
 
-- 500 Channel joins per second
 - 100 Channels per connected client
```
Comment (w3b6x9) [inline apps/docs/pages/guides/realtime/rate-limits.mdx:56]: @chasers these are both defaults so if you take out one you should take out the other. both of these can be customized so they don't technically apply to all projects and they currently don't apply to all projects.

**Review by gregnr — APPROVED** [review]

**Review by w3b6x9 — APPROVED** [review]

File: apps/docs/pages/guides/realtime/quotas.mdx [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+  title: 'Realtime Quotas',
+  description: 'Understanding Realtime quotas',
+  sidebar_label: 'Quotas',
+}
+
+<Admonition type="note">
+  Upgrade your plan to increase your quotas. Without a spend cap, or on an Enterprise plan, some
+  quotas are still in place to protect budgets. All quotas are configurable per project. [Contact
+  support](https://app.supabase.com/support/new) if you need your quotas increased. Our cluster
+  supports millions of concurrent connections and message throughput for production workloads.
```
Comment (kiwicopple) [inline apps/docs/pages/guides/realtime/quotas.mdx:14]:
```suggestion
  Upgrade your plan to increase your quotas. Without a spend cap, or on an Enterprise plan, some quotas are still in place to protect budgets. All quotas are configurable per project. [Contact support](https://app.supabase.com/support/new) if you need your quotas increased. Our cluster supports millions of concurrent connections and message throughput for production workloads.
```

File: apps/docs/pages/guides/realtime/quotas.mdx [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+</Admonition>
+
+## Quotas by Plan
+
+You can increase your quotas by upgrading your plan and removing your spend cap. We still have a starting quota set even without a spend cap to protect your budget just in case.
+
+If you need your quotas increased further please [contact support](https://app.supabase.com/support/new).
+
```
Comment (kiwicopple) [inline apps/docs/pages/guides/realtime/quotas.mdx:22]: consider representing as a table

|                          | Free | Pro | PAYG | Team | Enterprise |
|--------------------------|------|-----|------|------|------------|
| Concurrent clients       | 200  |     |      |      |            |
| Messages per second      |      |     |      |      |            |
| Channel joins per second |      |     |      |      |            |

File: apps/docs/pages/guides/realtime/quotas.mdx [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+For example, the [multiplayer.dev demo](/docs/guides/realtime/quickstart#cursor-positions) instantiates the Supabase client with an `eventsPerSecond` parameter.
+
+## Quota Errors
+
+When you reach a quota errors can appear in backend logs and messages in the WebSocket connection.
+
+<Admonition type="note">
+  Use the [Realtime Inspector](https://realtime.supabase.com/inspector/new) to reproduce an error
+  and share those connection details with Supabase support.
```
Comment (kiwicopple) [inline apps/docs/pages/guides/realtime/quotas.mdx:69]:
```suggestion
  Use the [Realtime Inspector](https://realtime.supabase.com/inspector/new) to reproduce an error and share those connection details with Supabase support.
```

File: apps/docs/pages/guides/realtime/quotas.mdx [hunk 10]
```diff
@@ -0,0 +1,112 @@
...
+
+Some quotas can cause a Channel join to be refused. Realtime will reply with one of the following WebSocket messages:
+
+- `too_many_channels`: Too many channels currently joined for a single client.
+- `too_many_connections`: Too many total concurrent connections for a project.
+- `too_many_joins`: Too many Channel joins per second.
+
+<Admonition type="note">
+  Use your browser's developer tools to find the WebSocket initiation request and view individual
+  messages.
```
Comment (kiwicopple) [inline apps/docs/pages/guides/realtime/quotas.mdx:92]:
```suggestion
  Use your browser's developer tools to find the WebSocket initiation request and view individual messages.
```

File: apps/docs/pages/guides/realtime/quotas.mdx [hunk 11]
```diff
@@ -0,0 +1,112 @@
...
+</Admonition>
+
+## Postgres Changes Payload Quota
+
+Realtime Postgres Changes has a change payload size quota of 1 mebibyte (MiB).
+
+When this quota is reached, the `new` and `old` record payloads only include the fields with a value size of less than or equal to 64 bytes.
+
+## Presence Quotas
```
Comment (kiwicopple) [inline apps/docs/pages/guides/realtime/quotas.mdx:96]: Any reason this isn't included in the "Quotas by Plan" section? Seems a bit hidden away

File: apps/docs/pages/guides/realtime/quotas.mdx [hunk 12]
```diff
@@ -0,0 +1,112 @@
...
+- `too_many_joins`: Too many Channel joins per second.
+
+<Admonition type="note">
+  Use your browser's developer tools to find the WebSocket initiation request and view individual
+  messages.
+</Admonition>
+
+## Postgres Changes Payload Quota
+
+Realtime Postgres Changes has a change payload size quota of 1 mebibyte (MiB).
```
Comment (kiwicopple) [inline apps/docs/pages/guides/realtime/quotas.mdx:92]:
```suggestion
Realtime Postgres Changes has a change payload size quota of 1 megabyte (MiB).
```

File: apps/docs/pages/guides/realtime/quotas.mdx (hunk 8 above)
Comment (w3b6x9) [reply apps/docs/pages/guides/realtime/quotas.mdx:19]: @kiwicopple good call.

@chasers i'm currently on docs and will update it to table.

File: apps/docs/pages/guides/realtime/quotas.mdx (hunk 12 above)
Comment (kiwicopple) [reply apps/docs/pages/guides/realtime/quotas.mdx:92]: Any reason this isn't included in the "Quotas by Plan" section? Seems a bit hidden away

File: apps/docs/pages/guides/realtime/quotas.mdx (hunk 8 above)
Comment (w3b6x9) [reply apps/docs/pages/guides/realtime/quotas.mdx:19]: @chasers for your review: https://github.com/supabase/supabase/pull/14946

**Review by gregnr — APPROVED** [review]

**Pull request merged**